tests/kustomize_build.sh

```diff
@@ -79,3 +65,26 @@ kpt fn source examples |
   kpt fn sink .
 assert_contains_string configmap_the-map.yaml "app: hello"
 assert_dir_exists knative-serving
+
+testcase "kpt_kustomize_build_declarative_git"
```

Comment:
nit: for clarity, highlight the test cases as such:

############################
testcasse blah
############################

Reply:
Having the testcase name as the first line is clear enough.